scri/asymptotic_bondi_data/bms_charges.py

```diff
@@ -30,30 +30,33 @@ def mass_aspect(self, truncate_ell=max):
         errors due to aliasing.)  If a callable is passed, it is passed on to the
         spherical_functions.Modes.multiply method.  See that function's docstring for details.
         The default behavior will result in the output having ell_max equal to the largest of
-        any of the individual Modes objects in the equation for \\Psi above -- but not the
+        any of the individual Modes objects in the equation for M above -- but not the
         product.
 
     """
     if callable(truncate_ell):
-        return self.psi2 + self.sigma.bar.eth_GHP.eth_GHP + self.sigma.multiply(self.sigma.bar.dot, truncator=truncate_ell)
+        return -(self.psi2 + self.sigma.multiply(self.sigma.bar.dot, truncator=truncate_ell))
```

Comment:
this is missing a ".real" at the end

Reply:
Thanks Keefe! Just pushed a fixup commit.